packages/common/src/util/constants/index.ts

```diff
@@ -2,6 +2,7 @@ export const STORAGE_PREFIX = '@signalwire:'
 export const ADD = 'add'
 export const REMOVE = 'remove'
 export const SESSION_ID = 'sessId'
+export const TIMEOUT_ERROR_CODE = -32000
```

Comment:
`-32000` is the same exact code for "Verto  - Authentication required" message.

Reply:
This seems like a problem no?

Reply:
Yes, it's a problem. I'm working on a workaround 👍